db/seeds.rb

```diff
@@ -21,21 +21,433 @@ def run_env_seeds!
+        short_title:'CEDAW'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Committee against Torture',
+        short_title:'CAT'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Subcommittee on Prevention of Torture',
+        short_title:'SPT'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Committee on the Rights of the Child',
+        short_title:'CRC'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Committee on Migrant Workers',
+        short_title:'CMW'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Committee on the Rights of Persons with Disabilities',
+        short_title:'CRPD'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:body,
+        title:'Committee on Enforced Disappearances',
+        short_title:'CED'
+      )
+
+    # TODO UN Sessions
+
+    # Human Rights (level 1 http://uhri.ohchr.org/search/annotations)
+    # TODO level 2 and 3 human rights
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'A General framework of implementation',
+        short_title:'A'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'B Universal or cross-cutting issues',
+        short_title:'B'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'D Civil & political rights',
+        short_title:'D'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'E Economic, social and cultural rights',
+        short_title:'E'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'F Specific persons or groups',
+        short_title:'F'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'G1 Members of minorities',
+        short_title:'G1'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'G3 Indigenous peoples',
+        short_title:'G3'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'G4 Migrants',
+        short_title:'G4'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'G5 Refugees & asylum seekers',
+        short_title:'G5'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'G6 Internally displaced persons',
+        short_title:'G6'
+      )
+    FactoryGirl.create(
+        :category,
+        taxonomy:right,
+        title:'H1 Human rights defenders',
+        short_title:'H1'
+      )
+
+
+      # Affected Persons (http://uhri.ohchr.org/search/annotations)
+      FactoryGirl.create(
+          :category,
+          taxonomy:persons,
+          title:'Children',
```

Comment:
@tmfrnz You are right, it would fill in the description.

You can get around that with this
`
description: '',
`